audit.py

```diff
@@ -56,27 +63,24 @@ def get_repos(organization, include_forked_repositories: bool, include_archived_
             yield repo
 
 
-def get_contributors(repo):
+def get_contributors(repo: Repository) -> NamedUser:
     """Return the non-bot contributors to a repository"""
     for contributor in repo.get_contributors():
         if contributor.type.lower() != "bot":
             yield contributor
 
 
-def get_members_and_membership(organization):
-    """Return the members of the organization."""
+def get_members_and_membership(organization) -> Tuple[NamedUser, Membership]:
```

Comment:
When running the script, I get an error originating here:
```console
$ python3 audit.py --help
Traceback (most recent call last):
  File "/Users/fniessink/Developer/github-audit/audit.py", line 73, in <module>
    def get_members_and_membership(organization) -> Tuple[NamedUser, Membership]:
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/typing.py", line 262, in inner
    return func(*args, **kwds)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/typing.py", line 896, in __getitem__
    params = tuple(_type_check(p, msg) for p in params)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/typing.py", line 896, in <genexpr>
    params = tuple(_type_check(p, msg) for p in params)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/typing.py", line 151, in _type_check
    raise TypeError(f"{msg} Got {arg!r:.100}.")
TypeError: Tuple[t0, t1, ...]: each t must be a type. Got <module 'github.NamedUser' from '/Users/fniessink/Developer/github-audit/venv/lib/python3.9/site-pac.
```

Reply:
Oops! Will fix.